dhcpd-formula/metadata/form.yml

```diff
@@ -86,6 +86,17 @@ dhcpd:
           $default:
           $help: Specify the URL of the initial boot file which is to be loaded by a client in EFI HTTP mode
           $optional: true
+        filename_efi_arm64:
+          $type: text
+          $visibleIf: .next_server != ''
+          $default: boot/grub-arm64.efi
+          $help: Specify the name of the initial boot file which is to be loaded by a client in EFI mode
+          $optional: true
+        filename_http_arm64:
+          $type: text
+          $default:
+          $help: Specify the URL of the initial boot file which is to be loaded by a client in EFI HTTP mode
+          $optional: true
```

Comment:
This is getting out of hand. Do we really need this in form? Also there are two other places where this can be setup in the form. Hosts config under subnet and hosts config outside of subnet.

Reply:
For Retail we are fine with defaults, but as long as this is a formula for generic use, under salt-formulas, I think it must be configurable.

Individual hosts have known architecture so they don't need multiple choices and existing fields are enough.